lib/xnvmec.c

```diff
@@ -969,6 +969,11 @@ static struct xnvmec_opt_attr xnvmec_opts[] = {
 		.descr = "If set, hint to use range as an integral unit when reading.",
 	},
 
+	{.opt = XNVMEC_OPT_KV_STORE_COMPRESS,
+	 .vtype = XNVMEC_OPT_VTYPE_NUM,
+	 .name = "compress",
+	 .descr = "KV Store Option to compress value on device"},
+
```

Comment:
The format of this is wrong, and it should be moved down under the rest of the KV options, so it resembles the order from libxnvmec.h:
```
const char *kv_key;
const char *kv_val;
bool kv_store_add;
bool kv_store_update;
bool kv_store_compress;
```

Reply:
It actually seems like it's duplicate of line 1015. Probably from bad rebase.